Jellyfin.Plugin.TubeArchivistMetadata/Tasks/JFToTubeArchivistProgressSyncTask.cs

```diff
@@ -167,7 +167,7 @@ public async Task ExecuteAsync(IProgress<double> progress, CancellationToken can
                                 foreach (Episode video in videos)
                                 {
                                     var videoYTId = Utils.GetVideoNameFromPath(video.Path);
-                                    var playbackProgress = _userDataManager.GetUserData(user.Id, video).PlaybackPositionTicks / TimeSpan.TicksPerSecond;
+                                    var playbackProgress = _userDataManager.GetUserData(user, video).PlaybackPositionTicks / TimeSpan.TicksPerSecond;
```

Comment:
`GetUserData` appears to have changed upstream so needs the user object rather than the ID